packages/coinstac-client-core/src/model-service.js

```diff
@@ -35,11 +36,13 @@ class ModelService {
     this.client = opts.client;
 
     /* istanbul ignore if */
-    if (!this.modelServiceHooks || !(this.modelServiceHooks instanceof Function)) {
+    if (!this.constructor.modelServiceHooks
```

Comment:
I typically break these up so that the boolean operator terminates the line and conditions are indented at equal levels. So:

```js
if (
  !this.constructor.modelServiceHooks ||
  !(this.constructor.modelServiceHooks instanceof Function)
) {
  // ...
}
```

As it turns out, two checks are redundant here. We don't need the lookup optimization before checking that `modelServiceHooks` is a function. So:

```js
if (!(this.constructor.modelServiceHooks instanceof Function)) {
  // ...
}
```